facturark/signer/signer.py

```diff
@@ -262,9 +266,10 @@ def _prepare_issuer_name(self, certificate_object):
         javax.security.auth.x500.X500Principal
         """
 
-        issuer_name = b','.join(
-            [key + b'=' + value for key, value in
-             reversed(certificate_object.get_issuer().get_components())])
+        issuer_name = ', '.join(
+            attr.rfc4514_string() for attr
+            in reversed(certificate_object.issuer._attributes)
+        ).encode('ascii')
```

Comment:
@tebanep I guessed `ascii` is the right byte encoding. Can you confirm? Should that go somewhere else? (eg. `encoder`)